pkgs/applications/window-managers/stumpish/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchurl, fetchFromGitHub, bash, gnused, ncurses, xorg, rlwrap }:
+
+stdenv.mkDerivation rec {
+  pname = "stumpish";
+  version = "0.0.1";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "stumpwm";
+    repo = "stumpwm-contrib";
+    rev = "9f5f06652c480159ec57d1fd8751b16f02db06dc";
+    sha256 = "1dxzsnir3158p8y2128s08r9ca0ywr9mcznivmhn1lycw8mg4nfl";
+  };
+
+  phases = [ "buildPhase" "installPhase" "postFixup" ];
```

Comment:
Manually setting phases is frowned upon https://github.com/NixOS/nixpkgs/issues/28910